frontend/src/pages/Interventions/FirstActivity.tsx

```diff
@@ -1,16 +1,34 @@
-import React, { ReactElement } from "react";
-import styled from "styled-components";
-import Blocker from "../../components/Blocker";
-import Layout from "../../components/Layout";
-import PromptSpeech from "../../components/PromptSpeech";
-import PurpleButton from "../../components/PurpleButton";
+import React, { ReactElement } from 'react';
+import styled from 'styled-components';
+import Blocker from '../../components/Blocker';
+import Layout from '../../components/Layout';
+import PromptSpeech from '../../components/PromptSpeech';
+import CloudImage from '../../components/CloudImage';
+import DelayedNextButton from '../../components/DelayedNextButton';
 import ReplayButton from '../../components/ReplayButton';
-import ExpandableImage from "../../components/ExpandableImage";
-import { SKY } from "../../constants/colors";
+import { SKY } from '../../constants/colors';
+import { connect } from 'react-redux';
+import { updateIntervention } from './data/actions';
+import { getNextActivityIdx } from '../../constants/utils';
+import ExpandableImage from '../../components/ExpandableImage';
 
 interface FirstActivityProps {
   title: string;
+  setId: string;
+  activityIdx: number;
+  wordIdx: number;
+  prompt: string;
   imageUrl: string;
+  maxWordLength: number;
+  updateIntervention: ({
+    setId,
+    wordIdx,
+    activityIdx,
+  }: {
```

Comment:
Maybe pass in a function that is literally `updateIntervention()`, and then the `Activities` DOM object will then do the updating of the index.